packages/modules/src/modules/epics/ContributionsEpicButtons.tsx

```diff
@@ -51,6 +71,8 @@ const PrimaryCtaButton = ({
     amountsTestName?: string;
     amountsVariantName?: string;
     numArticles: number;
+    showApplePayButton?: boolean;
+    submitComponentEvent?: (event: OphanComponentEvent) => void;
```

Comment:
Are the parameters `showApplePayButton` and `submitComponentEvent` needed/used in this function?

Reply:
Removed too